app/client/templates/views/dashboard.js

```diff
@@ -18,8 +18,8 @@ Template['views_dashboard'].helpers({
 
     @method (wallets)
     */
-    'wallets': function(disabled){
-        var wallets = Wallets.find({disabled: disabled}, {sort: {creationBlock: 1}}).fetch();
+    'wallets': function(){
+        var wallets = Wallets.find({disabled: false}, {sort: {creationBlock: 1}}).fetch();
```

Comment:
What's the advantage of showing a disabled wallet? I'd rather err on the side of not letting users accidentally send funds to unrecoverable address